dbms/src/Server/Client.cpp

```diff
@@ -445,12 +510,41 @@ class Client : public Poco::Util::Application
         return select(1, &fds, 0, 0, &timeout) == 1;
     }
 
+    const String prompt() const
+    {
+        String pattern = prompt_by_server_display_name;
+
+        for (const auto & [key, value]: colors)
+        {
+            String name = static_cast<String>(key);
+            String code = static_cast<String>(value);
+
+            boost::replace_all(pattern, "[" + name + "]",  "\33[1;" + code + "m");
+            boost::replace_all(pattern, "[/" + name + "]", "\033[0m");
+        }
+
+        std::map<String, String> environment = {
+            {"host",         config().getString("host", "localhost")},
+            {"port",         config().getString("port", "9000")},
+            {"display_name", server_display_name},
+            {"user",         config().getString("user",     "default")},
+            {"database",     config().getString("database", "default")},
```

Comment:
It is just default database. The user will expect changed prompt when current database is changed.